docs/how-to/contribute.md

```diff
@@ -0,0 +1,35 @@
+# How to contribute
+
+This document explains the processes and practices recommended for contributing enhancements to the Discourse operator.
```

Comment:
"Discourse" seems to have been copied and needs changing here.

Reply:
Ups. fixed